From: Andrea Bolognani <abologna@redhat.com>
Date: Thu, 18 Feb 2016 15:44:20 +0100
Subject: [PATCH] docs: List possible GIC versions

Recent changes to the handling of GIC version, specifically commit
2a7b11eafb67, have clearly defined what values are acceptable for the
version attribute of the <gic> element. Update the documentation
accordingly.

Origin: https://libvirt.org/git/?p=libvirt.git;a=commit;h=ff0e2ebde2a8d2443196fe010f3b93b8c189bdaf
Bug-Ubuntu: http://bugs.launchpad.net/bugs/1566564
Last-Updated: 2016-05-19

diff --git a/docs/formatdomain.html.in b/docs/formatdomain.html.in
index e96798f..a524c17 100644
--- a/docs/formatdomain.html.in
+++ b/docs/formatdomain.html.in
@@ -1578,8 +1578,9 @@
           For example, the 'aarch64' architecture uses
           <code>gic</code> instead of <code>apic</code>. The optional
           attribute <code>version</code> specifies the GIC version;
-          however, it may not be supported by all hypervisors. <span
-          class="since">Since 1.2.16</span>
+          however, it may not be supported by all hypervisors. Accepted
+          values are <code>2</code>, <code>3</code> and <code>host</code>.
+          <span class="since">Since 1.2.16</span>
       </dd>
     </dl>
 
-- 
2.8.1

